onboard-jmuniz/.nvmrc

```diff
@@ -0,0 +1 @@
+20.17.0
```

Comment:
é sempre bom deixar uma linha a mais, se vc der uma olhadinha nos files changed fica um (-) vermelhão, dai vc vai saber que é só questão de eol (end of line)